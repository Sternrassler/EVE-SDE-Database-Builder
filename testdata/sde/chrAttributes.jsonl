{"attributeID":164,"attributeName":"charisma","description":"Charisma of the character.","iconID":21,"notes":"Social skills","shortDescription":"Social ability"}
{"attributeID":165,"attributeName":"intelligence","description":"Intelligence of the character.","iconID":21,"notes":"Learning skills","shortDescription":"Mental capacity"}
{"attributeID":166,"attributeName":"memory","description":"Memory of the character.","iconID":21,"notes":"Training speed","shortDescription":"Knowledge retention"}
