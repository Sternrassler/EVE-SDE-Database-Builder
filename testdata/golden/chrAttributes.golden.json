[
  {
    "attributeID": 164,
    "attributeName": "charisma",
    "description": "Charisma of the character.",
    "iconID": 21,
    "shortDescription": "Social ability",
    "notes": "Social skills"
  },
  {
    "attributeID": 165,
    "attributeName": "intelligence",
    "description": "Intelligence of the character.",
    "iconID": 21,
    "shortDescription": "Mental capacity",
    "notes": "Learning skills"
  },
  {
    "attributeID": 166,
    "attributeName": "memory",
    "description": "Memory of the character.",
    "iconID": 21,
    "shortDescription": "Knowledge retention",
    "notes": "Training speed"
  }
]